### PR TITLE
Replaces the limb grower with a box of surplus limbs

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: The limb grower has been replaced with a box of surplus limbs. Visit robotics or harvest limbs from another person if you want quality.
/:cl:

I think the limb grower is a very boring "quality of life" machine that erases any sort of consequence from losing body parts.  Getting robotic limbs at least leaves a kind of record of the incident on your character as the round goes on.

The surplus limbs are cheap and break easily, encouraging people to visit robotics for actual cyborg limbs, or to create freaky Frankenstein monsters by harvesting limbs from other people.

### CODE REVIEW

**Comment by TehZombehz** [comment]
Can't we have both?

**Comment by ExcessiveUseOfCobblestone** [comment]
> not harvesting the parts of suicides

**Comment by Incoming5643** [comment]
To be honest if the limb grower was made build-able it could be a decent mid-tier research reward.

**Comment by PKPenguin321** [comment]
Pretty sure it already is buildable.

**Comment by dannno** [comment]
>Can't we have both?

what would the point be then

**Comment by Iamgoofball** [comment]
replace both of these with limbplants from goon

**Comment by Aloraydrel** [comment]
Limb harvesting suicides/cloned people is the most fun you can have as a """Doctor"""

**Comment by kevinz000** [comment]
what if i want natural limbs you bully ;-;7

**Comment by KorPhaeron** [comment]
@kevinz000 

Don't lose them in the first place

**Comment by PKPenguin321** [comment]
one time i swooped up a dying one armed botanist from out of a plasma fire and went to the holodeck but unfortunately it was breached so we couldnt use it. i put him on the big 3x2 table in the dorms then ran out and stole the axe from the bridge and dragged in a husked out corpse from the fire. the shuttle was on its way so i used the axe to chop off the husks arm, grabbed a bedsheet from a nearby dorm room, and used the tools in my toolbelt + the axe to stitch the dead zombie husk arm to the botanist in an amazing feat of ghetto surgery while also racing against the shuttle. after it was finally on his stump i plopped him in the holodeck sleeper and just barely was able to revive him back to full health, then i pulled him and dashed through the fire and flames to the shuttle with only seconds to spare, grabbing an extinguisher to put us out along the way.

dismemberment is cool and having it be made pointless by this limb grower meme machine is lame. 👍

**Comment by Haevacht** [comment]
I'm with incoming. Make it a probably hard research reward, keep stock disposable limbs.

Also peg legs, remie i think told someone in some thread how to peg leg.

**Comment by XDTM** [comment]
100% fine with it, if there's a particular need for limbs they can ask rnd to make one. We'll see less traitor armblades though.

**Comment by PKPenguin321** [comment]
Funny you mention that @Haevacht after I saw this pr I wrote some code for peg legs that I might PR later if I get around to finishing it (the sprites are fucking awful though because I made them)

**Comment by Arianya** [comment]
I don't see how the limb grower machine made dismemberment "pointless", considering finding someone willing let alone able to do the surgery is a minor miracle, especially if the station falls into chaos.

If proper limb replacements are going to be the realm of the Roboticist anyway, why not just give him a cautery so he can do proper surgery and remove the often entirely unused Surgery room/Observation area?

**Comment by Jalleo** [comment]
We should remap actually.

**Comment by KorPhaeron** [comment]
@Haevacht 

This is basically peg legs but sci-fi and I didn't have to make new sprites

@Arianya 

Not pointless, but it does erase the damage entirely. This "scars" your character and adds a bit of flavour and story to them if they're sporting a prosthetic or lizard limb instead of looking how they did before the trauma

File: code/modules/surgery/bodyparts/robot_bodyparts.dm [hunk 1]
```diff
@@ -158,3 +158,34 @@
...
+	icon = 'icons/mob/augments.dmi'
+	icon_state = "surplus_l_leg"
+	max_damage = 20
+
+/obj/item/bodypart/r_leg/robot/surplus
+	name = "surplus prosthetic leg"
+	desc = "A skeletal, robotic limb. Outdated and fragile, but it's still better than nothing."
+	icon = 'icons/mob/augments.dmi'
+	icon_state = "surplus_r_leg"
+	max_damage = 20
```
Comment (coiax) [inline code/modules/surgery/bodyparts/robot_bodyparts.dm]: These should be craftable, somehow. Maybe order a crate of surplus limbs from cargo?

**Comment by Cyberboss** [comment]
How about just removing the starter fleshwhatever reagent that it comes with?

**Comment by chickenboy10** [comment]
I know its a bit late for input considering this has already been pulled but since I made the limbgrower I thought I would give my thoughts on it.
I originally designed the limbgrower to be a way to lessen the load on roboticists and make surgery more of a medical doctor role. I had found it strange that on occasion when I walked into medbay and asked for surgery, most medical doctors would shrug and say "I dont know how to do it".
I think, the reasoning for this is just as Arianya pointed out, since roboticists had all the limbs and augmentations, they did all the surgery, stopping medical doctors from ever getting to learn or perform it in medbay.
I was happy to see that, when I played, it was used occasionally when doctors realised that they actually could attempt to create limbs for people, instead of having to send them off to robotics.
I feel as if this PR might again push people towards robotics, overloading them with work that should be shared with medbay, but I'll try to be optimistic as this should still in theory give doctors something to work with.

**Comment by Cyberboss** [comment]
Remove all surgery tools from roboticist other than saw

**Pull request merged**